### PR TITLE
Add CLI/API docs for service

### PR DESCRIPTION
**Pull request by feiskyer**
cc/ @Gnep @gnawux @carmark

### CODE REVIEW

File: Feature/service.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+As an example, consider an image-processing backend which is running with 3 replicas. Those replicas are fungible - frontends do not care which backend they use. While the actual containers that compose the backend set may change, the frontend clients should not need to be aware of that or keep track of the list of backends themselves. The Service abstraction enables this decoupling.
+
+Each service will receive an internal IP address which can only be accessed internal. For external accessing, a floating IP address need to be attached by `attach-fip` command.
+
+### Examples
+
+**Create a http service:**
+
+    $ service create --service-port=80 --label=app=nginx --image=nginx --replicas=3 http
```
Comment (carmark) [inline Feature/service.md]: missing `hyper`

File: Reference/CLI/service_create.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    -v, ---volume=[]             Volume for each container
+    -w, --workdir                Working directory inside the container
+
+Creates a new service based on the image and options.
+
+### Examples
+
+**Create a http service:**
+
+    $ service create --service-port=80 --label=app=nginx --image=nginx --replicas=3 http
```
Comment (carmark) [inline Reference/CLI/service_create.md]: missing `hyper`

Comment (feiskyer) [inline Reference/CLI/service_create.md]: already fixed in previous commit.

File: Reference/CLI/service_rm.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+**Remove a service and all backend containers belonging to it:**
+
+    $ hyper service rm http
+    http
+
+**Remove a service but keep backend containers running:**
+
+    $ hyper service rm --keep http
+    service1
```
Comment (carmark) [inline Reference/CLI/service_rm.md]: should be `http`?

Comment (feiskyer) [inline Reference/CLI/service_rm.md]: ack

**Comment by Gnep** [comment]
@feiskyer 

Plz re-submit a PR based on `develop` branch, I will review and edit that.

**Comment by feiskyer** [comment]
@Gnep See new PR at #48.

**Pull request closed without merging**